contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -329,6 +330,40 @@ CREATE OR REPLACE VIEW information_schema_tsql.columns AS
 									'SELECT, INSERT, UPDATE, REFERENCES'))
 		AND ext.dbid =sys.db_id();
 
+GRANT SELECT ON information_schema_tsql.columns_internal TO PUBLIC;
```

Comment:
We don't need to grant perms to public if it is for internal use only.

Reply:
removed